src/settings/MappingProfiles/detailsSections/edit/InvoiceDetailSection/index.js

```diff
@@ -0,0 +1,7 @@
+export * from './InvoiceInformation';
+export * from './InvoiceAdjustments';
+export * from './VendorInformation';
+export * from './ExtendedInformation';
+export * from './InvoiceLineInformation';
+export * from './InvoiceLineFundDistribution';
+export * from './InvoiceLineAdjustments';
```

Comment:
Set exports in alphabetical order, please.

Reply:
done